test/functional/tests/failover_standby/test_cache_activation_perf.py

```diff
@@ -0,0 +1,151 @@
+                                   cache_mode=CacheMode.WB,
+                                   cache_line_size=CacheLineSize.LINE_4KiB)
+
+        cache.add_core(core_device)
+        current_cache_id = cache.cache_id
+
+    with TestRun.step("Disable seq. cutoff, cleaning policy and udev"):
+        Udev.disable()
+        cache.set_seq_cutoff_policy(SeqCutOffPolicy.never)
+        cache.set_cleaning_policy(CleaningPolicy.nop)
+
+    with TestRun.step("Fill the cache with dirty data"):
+
+        fill_size = cache.get_statistics().config_stats.cache_size * CacheLineSize.LINE_4KiB
+        (
+            Fio()
+            .create_command()
+            .io_engine(IoEngine.libaio)
+            .block_size(Size(4, Unit.KiB))
+            .read_write(ReadWrite.write)
+            .target(core_device)
+            .size(fill_size)
+            .direct()
+            .run()
+        )
+
+    with TestRun.step("Check usage statistics"):
+        stats = cache.get_statistics(percentage_val=True)
+        if stats.usage_stats.dirty < 98 or stats.usage_stats.occupancy < 98:
+            TestRun.fail(f"Cache is not pre-filled correctly, "
+                         f"dirty={stats.usage_stats_dirty}%, "
+                         f"occupancy={stats.usage_stats.occupancy}%")
+
+    with TestRun.step("Dump metadata from the active cache device"):
+        md_size = test_drive_size - fill_size
+        dump_file_path = "/tmp/test_cache_activation_perf.dump"
+        md_dump = File(dump_file_path)
+        md_dump.remove(force=True, ignore_errors=True)
+
+        dd_count = int(md_size / Size(1, Unit.MebiByte)) + 1
+        (
+            Dd()
+            .input(cache_devices[0].partitions[0].path)
+            .output(md_dump.full_path)
+            .block_size(Size(1, Unit.MebiByte))
+            .count(dd_count)
+            .run()
+        )
+        md_dump.refresh_item()
+
+    with TestRun.step("Stop the cache without flushing."):
+        cache.stop(no_data_flush=True)
+
+    with TestRun.step("initialize standby instance with cls = 4k"):
+        standby_cache = casadm.standby_init(cache_dev=cache_devices[1],
+                                            cache_line_size=CacheLineSize.LINE_4KiB,
+                                            cache_id=current_cache_id)
+
+    with TestRun.step("verify cache exported object has appeared"):
+        output = TestRun.executor.run_expect_success(f"ls -la /dev/ | grep cas-cache-1")
+        if output.stdout[0] != "b":
+            TestRun.fail("The cache exported object is not a block device")
+
+    with TestRun.step("Copy valid metadata to the standby cache exp. obj."):
+        dd_count = int(md_size / Size(1, Unit.MebiByte)) + 1
+        (
+            Dd()
+            .input(md_dump.full_path)
+            .output("/dev/cas-cache-1")
+            .block_size(Size(1, Unit.MebiByte))
+            .count(dd_count)
+            .run()
+        )
+
+    with TestRun.step("Detach the standby cache"):
```

Comment:
I don't see `standby_detach in this step, but I see `standby_activate`, something's missing?

Reply:
fixed